src/bin/pg_autoctl/pgctl.c

```diff
@@ -1424,7 +1419,10 @@ pg_setup_standby_mode(uint32_t pg_control_version,
 	 */
 	if (pg_control_version >= 1100)
 	{
-		if (!pg_receivewal(pgdata, pg_ctl, replicationSource, "0/1", LOG_DEBUG))
+		NodeAddress *primaryNode = &(replicationSource->primaryNode);
```

Comment:
I think we should only be verbose about it when it failed, and refrain from saying anything to the INFO logs when it worked, so that's why we have the current level of verbosity...